core/client/queries/get-cart.ts

```diff
@@ -136,8 +233,12 @@ export const getCart = cache(async (cartId?: string, channelId?: string) => {
     },
     channelId,
   });
+};
+
+export const getCart = cache(async (cartId?: string, channelId?: string) => {
+  const response = await fetchCart(cartId, channelId);
 
-  const cart = response.data.site.cart;
```

Comment:
This change seems unnecessary. Can we just destructure here instead of splitting the function in two separate functions? I understand you are trying to fetch the cart separately, but it looks like you just need the carts currency code. I would just create a separate query for the currency code, more inlined with the route handler that needs it.
```ts
const { cart, checkout } = response.data.site; 
```

Reply:
I will separate this this function, thanks